sdk/samples/iot/docs/how_to_iot_hub_samples_linux.md

```diff
@@ -44,51 +46,53 @@ To run the samples, ensure you have the following programs and tools installed o
 
     ```bash
     sudo apt-get update
-    sudo apt-get install build-essential # make and gcc
-    sudo apt-get install curl unzip tar pkg-config
-    sudo apt-get install git
-    sudo apt-get install openssl libssl-dev
+    sudo apt-get install build-essential curl unzip tar pkg-config git openssl libssl-dev
     ```
 
 ## Setup Instructions
 
-1. Install Microsoft [vcpkg](https://github.com/microsoft/vcpkg) package manager and [Eclipse Paho MQTT C client](https://www.eclipse.org/paho/). This installation may take an extended amount of time (~20-30 minutes).
+1. Install Microsoft [vcpkg](https://github.com/microsoft/vcpkg) package manager and [Eclipse Paho MQTT C client](https://www.eclipse.org/paho/). This installation may take an extended amount of time (~15-20 minutes).
 
     ```bash
-    cd / # Run this command from any directory to go to the root.
-    /$ sudo git clone https://github.com/Microsoft/vcpkg.git
-    /$ cd vcpkg
-    /vcpkg$ sudo ./bootstrap-vcpkg.sh
-    /vcpkg$ sudo ./vcpkg install --triplet x64-linux curl cmocka paho-mqtt
-    /vcpkg$ cd ..
+    cd ~ # Run this command from any directory to go to your user home directory.
+    ~$ sudo git clone https://github.com/Microsoft/vcpkg.git
+    ~$ cd vcpkg
+    ~/vcpkg$ sudo ./bootstrap-vcpkg.sh
+    ~/vcpkg$ sudo ./vcpkg install --triplet x64-linux curl cmocka paho-mqtt
+    ~/vcpkg$ cd ..
     ```
 
 2. Set the vcpkg environment variables.
 
     ```bash
-    /$ export VCPKG_DEFAULT_TRIPLET=x64-linux
-    /$ export VCPKG_ROOT=/vcpkg
+    ~$ export VCPKG_DEFAULT_TRIPLET=x64-linux
+    ~$ export VCPKG_ROOT=~/vcpkg
     ```
 
     NOTE: Please keep in mind, **every time a new terminal is opened, the environment variables will have to be reset**.
 
-3. Install the latest version of [CMake](https://cmake.org/download).  In this sample, it is `cmake-3.18.2.tar.gz`. This installation may also take an extended amount of time (~20-30 minutes).
+3. Install [CMake](https://cmake.org/files).
 
-    Move the downloaded tar file to the root directory. Then run the following commands:
+    For Ubuntu 18.04 or 20.04, run the following command:
+
+    ```
+    sudo apt-get install cmake
+    ```
+
+    For Ubuntu 16.04, run the following commands:
 
     ```bash
-    /$ sudo apt-get purge cmake
-    /$ sudo tar -xvzf cmake-3.18.2.tar.gz # Use latest version
-    /$ cd cmake-3.18.2
-    /cmake-3.18.2$ sudo ./bootstrap && sudo make && sudo make install
-    /cmake-3.18.2$ cd ..
+    ~$ wget https://cmake.org/files/v3.18/cmake-3.18.3-Linux-x86_64.sh # Use latest version.
```

Comment:
We have a very similar set of instructions on `sdk/samples/iot/README.md`.  I haven't stared at docs/flow enough to know for sure but is there a way we could consolidate this?

Part of having duplicate info is that isntructions start to diverge.  In other example user is pointed to `latest-cmake-version` whereas here its hard-coded to 3.18.  FWIW if we need both samples, I think just pointing user to 3.18 is right call.

Reply:
The point of the how-to guide is to be very, very specific step-by-step example of setting up the samples, so a lot is duplicated.  That's going to be the nature of it.

Reply:
I can update the main README to use 3.18.3 with the comment to use latest version.  This way it is an easier copy paste for the user.